I1724 ibm style guide (#1221)
codenvy/che-docs: src/main/pages/che-7/end-user-guide/ref_devfile-reference.adoc

* Omit the leading slash (`my-module/`) to create all `my-module` directories that exist in the project. Including, for example, `/addons/my-module/`.
+

> The trailing slash indicates that only directories with the given name (including their content) will be created.

Reviewer: ```suggestion
The trailing slash indicates that only directories with the given name (including their content) are created.
```

+
WARNING: Listening on any other interface than the local loopback poses a security risk because such server is accessible without the JWT authentication within the cluster network on the corresponding IP addresses.

* `path`: The URL of the endpoint.

> * `unsecuredPaths`: A comma-separated list of the unsecured endpoint paths, staying as unsecured even if the `secure` attribute is set to `true`.

Reviewer: Confusing.
```suggestion
* `unsecuredPaths`: A comma-separated list of endpoint paths that are to stay unsecured even if the `secure` attribute is set to `true`.
```

=== {prod-short}-specific commands

> Each {prod-short}-specific command has an *action* attribute that is a command to execute and a *component* attribute that specifies the container for the command execution. The commands are run using the default shell in the container.

Reviewer: ```suggestion
Each {prod-short}-specific command features:

 * An `action` attribute that is a command to execute.
 * A `component` attribute that specifies the container in which to execute the command.
 
 The commands are run using the default shell in the container.
```

==== Setting the default way of opening preview URLs

By default, a notification that asks the user about the URL opening preference is displayed. 

To specify the preferred way of previewing a service URL:

. Open {prod-short} preferences in *File -> Settings -> Open Preferences* and find `che.task.preview.notifications` in the *{prod-short}* section.

. Choose from the list of possible values:
+

> * `on` -- enables a notification that ask the user about the URL opening preferences

Reviewer: ```suggestion
* `on` -- enables a notification for asking the user about the URL opening preferences
```

[id="attribute-editorfree_{context}"]
=== Attribute: editorFree

> When an editor is not specified in a devfile, a default is provided. When no editor is needed, the `editorFree` attribute is applied. The default value is `false`, and it means that the devfile needs the default editor to be provisioned.

Reviewer: Change in meaning.
```suggestion
When an editor is not specified in a devfile, a default is provided. When no editor is needed, use the `editorFree` attribute. The default value of `false` means that the devfile requests the provisioning of the default editor.
```

=== Attribute: persistVolumes (ephemeral mode)

> By default, volumes and PVCs specified in a devfile are bound to a host folder to persist data even after a container restart. Sometimes, it may be necessary to disable data persistence, such as when volume back end is slow, and it is needed to make the workspace faster. To achieve it, the `persistVolumes` devfile attribute must be changed. The default value is `true`, and in case of `false`, `emptyDir` volumes will be used for configured volumes and PVC.

Reviewer: ```suggestion
By default, volumes and PVCs specified in a devfile are bound to a host folder to persist data even after a container restart. To disable data persistence to make the workspace faster, such as when the volume back end is slow, modify the `persistVolumes` attribute in the devfile. The default value is `true`. Set to `false` to use `emptyDir` for configured volumes and PVC.
```
